client/src/components/related/ProductCompare.jsx

```diff
@@ -33,21 +48,35 @@ class ProductCompare extends React.Component {
   //   }
   // }
 
-  // handleDragStart(event) {
-  //   // event.stopPropagation();
-  //   let x = event.nativeEvent.offsetX;
-  //   let y = event.nativeEvent.offsetY;
+  handleDragStart(event) {
+    // event.preventDefault();
+    this.offsetX = event.nativeEvent.offsetX;
```

Comment:
is `nativeEvent` a more mobile-combatible way to get this? I currently use `e` or `window`, but wonder if a future improvement is to handle the mobile case, as those might not be reliable...